<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="fds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Patient</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Recherche Patient :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Nom :</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight|Qt::AlignVCenter">
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="text">
                <string>Boulanger</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Normal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>7</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Prénom :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="text">
                <string>Alex</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>ID :   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="text">
                <string>26</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit"/>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Rechercher</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Personnel soins</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>9</y>
        <width>511</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionPatient"/>
     <addaction name="actionPersonnel_de_soins"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPatient"/>
   <addaction name="separator"/>
   <addaction name="actionPersonnel_de_soins"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionPatient">
   <property name="icon">
    <iconset resource="../polytech.qrc">
     <normaloff>:/Pictures/polytech tours.gif</normaloff>:/Pictures/polytech tours.gif</iconset>
   </property>
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionPersonnel_de_soins">
   <property name="text">
    <string>Personnel de soins</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>à propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../polytech.qrc"/>
 </resources>
 <connections/>
</ui>
